connectors/services/sync.py

```diff
@@ -164,3 +148,27 @@ async def _run(self):
                 await self.sync_job_index.close()
             await es.close()
         return 0
+
+    async def _prepare_connector(self, connector):
```

Comment:
What do you think of adding a docstring for this method @wangch079?

Reply:
This method `_prepare_connector` just calls `connector.prepare()`, catches all the possible exceptions, and then returns `True` or `False` indicating whether this operation is successful.

I'm not sure whether I should add `docstring` to a **private** method, so I updated the `docstring` of `Connector.prepare()` method.